### PR TITLE
Book-Library features

### PR DESCRIPTION
**Pull request by uche-inyama**
Hi TSE, :wave: 
In this project we also implemented **``` local storage```**. Consequently, we did not declare the array globally;  you will find the array in getLibrary() method, line 39.

Thanks in advance. :slightly_smiling_face:

### CODE REVIEW

**Comment by shloch** [comment]
### PR under review by SHEY LOUIS CHIA

File: index.html [hunk 1]
```diff
@@ -4,7 +4,82 @@
...
+              value=""
+              placeholder="Who's the author"
+            /><br />
+            <label>Number of pages: </label><br />
+            <input
+              class="field"
+              type="number"
+              name="pages"
+              placeholder="No of pages"
+            /><br />
```
Comment (shloch) [inline index.html]: Hello great job for your project, please look for a way to control the submission of an empty form. "Empty" books can be added. You either add the option `required` to these inputs, or you control that with javascript

**Review by shloch — CHANGES_REQUESTED** [review]
### Please fix the minor issue above and file for a Re-review 
### Happy coding 😊😊

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
# Changes required 🙁 

The book is not being added to the library as it is throwing a validations error.
```
Please check, a required field(s) are empty
```
Pls check this and request another review.

James Okunlade

**Review by SunnyAsar — APPROVED** [review]
# Hello @uche-inyama  and @IBTechRaj 👋
## Looking Good ✨  Please proceed

# You May Pass 👍

**Pull request merged**